lib/app/item.ex

```diff
@@ -69,6 +80,7 @@ defmodule App.Item do
     Item
     |> Repo.get!(id)
     |> Repo.preload(tags: from(t in Tag, order_by: t.text))
+    |> Repo.preload(lists: from(l in L, order_by: l.name))
```

Comment:
This query is essentially saying: When fetching an `item` _always_ fetch ***all*** of the associated lists. 🤷‍♂️
This is an inefficient query because we aren't going to _display_ multiple `lists` at once in the UI.
Instead what we want is to select a `list` and then fetch all the `items` associated with it.

Reply:
At the moment I'm displaying the item's lists:
![image](https://user-images.githubusercontent.com/6057298/194835551-9448c516-1f2c-4d51-a60b-cae41efdb974.png)

We can modify the UI/query to not have this, although I don't think the preload will slow down the queries but we can definitely remove it if it become an issue :+1: